armi/physics/neutronics/fissionProductModel/tests/test_lumpedFissionProduct.py

```diff
@@ -98,55 +101,26 @@ def setUp(self):
             io.StringIO(LFP_TEXT)
         )
 
-    def test_setGasRemovedFrac(self):
-        """Test of the set gas removal fraction"""
-        lfp = self.fpd.createSingleLFPFromFile("LFP38")
-        xe135 = nuclideBases.fromName("XE135")
-        gas1 = lfp[xe135]
-        lfp.setGasRemovedFrac(0.25)
-        gas2 = lfp[xe135]
-        self.assertAlmostEqual(gas1 * 0.75, gas2)
-
     def test_getYield(self):
         """Test of the yield of a fission product"""
         xe135 = nuclideBases.fromName("XE135")
         lfp = self.fpd.createSingleLFPFromFile("LFP39")
-        lfp[xe135] = 3
+        lfp[xe135] = 1.2
         val3 = lfp[xe135]
-        self.assertEqual(val3, 3)
-        self.assertIsNone(lfp[5])
-
-    def test_getNumberFracs(self):
-        xe135 = nuclideBases.fromName("XE135")
-        lfp = self.fpd.createSingleLFPFromFile("LFP38")
-        numberFracs = lfp.getNumberFracs()
-        self.assertEqual(numberFracs.get(xe135), 1.0)
+        self.assertEqual(val3, 1.2)
+        self.assertEqual(lfp[5], 0.0)
```

Comment:
This is changed because it used to return a yield of `None` for a nuclide/key that didn't exist. This doesn't really make sense because `0.0` is physical.